sdk/cosmos/azure-spring-data-cosmos/pom.xml

```diff
@@ -100,11 +100,6 @@
       <artifactId>jackson-datatype-jdk8</artifactId>
       <version>2.13.4</version> <!-- {x-version-update;com.fasterxml.jackson.datatype:jackson-datatype-jdk8;external_dependency} -->
     </dependency>
-    <dependency>
```

Comment:
what about the two jackson dependencies above?` jackson-module-parameter-names` and `jackson-datatype-jdk8`

Reply:
Those are not part of azure-core or any of the libraries that spring-data-cosmos inherits. That's why had to leave them there as it is. Just removed the commons ones (coming from azure-core).